app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+class CommandError(Exception):
+    pass
+
+
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
+
+    if command_list[0] == "cp":
```

Comment:
Task states that in addition ot == "cp" you gotta check if command string you received has only 3 elements, and also if file_in and file_out are the same function should do nothing.